tasks/10_checks-repos.yml

```diff
@@ -0,0 +1,123 @@
+---
+## file checks_repos.yml of AlternC role
+## Check of AlternC requiremnts (acl and quotas)
+## set up of alternc and other needed repos
+
+- name: Check if alternc_debconf_alternc_html's partition has ACL and grpquota active
+  include_tasks: include/html-mount.yml
+  when: mount.mount is defined
+  loop: '{{ ansible_facts.mounts }}'
+  loop_control:
+    loop_var: mount
+
+- name: Set query to check AlternC html mount options
+  set_fact:
+    alternc_html_mount_query: "[?mount=='{{ alternc_html_mount }}'].options"
+
+- name: ACL of alternc www directory mount
+  fail:
+    msg: >
+        The partition mounted where will be created the html directory for AlternC has no ACL. 
+        Such ACL is required by AlternC. Sorry, we stop the installation. Try again building 
+        first a host which has ACL activated on the partition where AlternC html files will be
+        installed (by default /var/www/alternc).
+  when: ansible_facts.mounts | json_query( alternc_html_mount_query ) | first | regex_search('noacl')
+
+- debug: 
+    msg: "{{ ansible_facts.mounts | json_query( alternc_html_mount_query ) | first }}"
+    verbosity: 2
+
+- name: Group quota on alternc www directory mount
+  pause:
+    prompt: >
+        The partition mounted where will be created the html directory for AlternC (by default 
+        /var/www/alternc) has no group quota management. You won't be able to manage AlternC users' quotas. 
+        Do you want to continue?
+  when: not ansible_facts.mounts | json_query( alternc_html_mount_query ) | first | regex_search('(grpjquota=aquota\.group|grpquota)')
+
+
+- name: When we want to set a favicon.ico, check if icoutils is installed
+  block: 
+
+  - name: Gather the list of local packages
+    local_action: package_facts
```

Comment:
When I run from my computer the role fails with this:

> Could not detect a supported package manager from the following list: ['portage', 'pacman', 'rpm', 'pkg', 'apt'], or the required Python library is not installed. Check warnings for details.

I solved it, installing the `python-apt` package and running it again

Reply:
Maybe we should check this package in the beginning of the role.

Reply:
Trying to sort out this problem, I see that all this block concerns the local controller, and the whole is to have icoutils, to build the favicon, a task that is not really part of AlternC installation. Morover, that `python-apt` is a specific case for a Debian-like local OS, if we want a role agnostic on controller's OS (as we did using "package" module) we should test the local OS before installing, and better treat other OS cases. 

I propose we cosider this block of tasks as a plus, to easily set a favicon from a png image, as far as operator's controllers complies. If not, sorry: build your own favicon and set the appropriate data to omit this collateral benefit. 

If you agree, I ensure it's documented and works as such.

Reply:
Yes, I agree, we have to make all that block of tasks optional, it will be the best, python-apt can be installed previously (indicate this as a requirement for this plus function).

Reply:
I addressed this conversation in commit 22cd438: 
- I coded a better condition for the icoutils tasks block including omitting it if the favicon file provided is a .ico
- I documented the way to manually omit the block (use of a tag)